android/src/com/eegeo/mobilesdkharness/BackgroundThreadActivity.java

```diff
@@ -33,6 +51,24 @@ public void onCreate(Bundle savedInstanceState)
 		m_surfaceView = (EegeoSurfaceView)findViewById(R.id.surface);
 		m_surfaceView.getHolder().addCallback(this);
 		m_surfaceView.setActivity(this);
+		
```

Comment:
There seems to be a lot of VR specific change in here. Can we either/and/or separate some of it out into other classes? Maybe a module?